Source/SwiftLintFramework/Reporters/JUnitReporter.swift

```diff
@@ -11,11 +11,12 @@ struct JUnitReporter: Reporter {
     static func generateReport(_ violations: [StyleViolation]) -> String {
         let warningCount = violations.filter({ $0.severity == .warning }).count
         let errorCount = violations.filter({ $0.severity == .error }).count
+        let testCount = violations.count
```

Comment:
```suggestion
        let testCount = warningCount + errorCount
```
So separate issues always add up to the total number.